packages/loader/container-utils/src/error.ts

```diff
@@ -166,8 +166,9 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
         };
 
         const normalizedError = normalizeError(originalError, { props });
-
-        if (isExternalError(normalizedError) || isNormalizedLoggingError(normalizedError)) {
+        // Note that other errors may have the NORMALIZED_ERROR_TYPE errorType,
+        // but if so they are still suitable to be wrapped as DataProcessingError.
+        if (isExternalError(normalizedError) || normalizedError.errorType === NORMALIZED_ERROR_TYPE) {
```

Comment:
Pretty sure you can drop `isExternalError` check here altogether now.  Any external error would have gotten that "genericError" errorType when normalized on line 168.  See if your unit tests still pass without it :)